gas/doc/c-riscv.texi

```diff
@@ -47,3 +47,89 @@ the soft-float calling convention.
 
 @end table
 @c man end
+
+@node RISC-V-Directives
+@cindex Machine directives
+
+@table @code
+
+@cindex @code{align} directive
+@item .align @var{size-log-2}
+Align to the given boundary, with the size gives as log2 the number of bytes to
+align to.
+
+@cindex Data directives
+@item .half @var{value}
+@itemx .word @var{value}
+@itemx .dword @var{value}
+Emits a half-word, word, or double-word value at the current position.
+
+@cindex DTP-relative data directives
+@item .dtprelword @var{value}
+@itemx .dtpreldword @var{value}
+Emits a DTP-relative word (or double-word) at the current position.
+
+@cindex BSS directive
+@item .bss
+Sets the current section to the BSS section.
+
+@cindex LEB128 directives
+@item .uleb128 @var{value}
+@itemx .sleb128 @var{value}
+Emits a signed or unsigned LEB128 value at the current position.
+
+@cindex Option directive
+@item .option @var{argument}
+Modifies RISC-V specific assembler options in-line with the assembly code.
+This is used when particular instruction sequences must be assembler with a
+specific set of options.  For example, since we relax addressing sequences to
+shorter GP-relative sequences when possible the initial load of GP must not be
+relaxed and should be emitted as something like
+
+.option push
+.option pop
```

Comment:
is this first `pop` correct?

Reply:
Fixed in the patch I commited.  See
https://sourceware.org/ml/binutils/2017-12/msg00021.html